src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java

```diff
@@ -113,20 +113,20 @@ private void testActions() {
 
         ______TS("Typical case: no picture");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Singaporean",
-                                         "male", "this is enough!$%&*</>");
+        profilePage.editProfileThroughUi("short.name", "e@email.tmt", "inst", "Singaporean", "male",
+                                         "this is enough!$%&*</>");
         profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Singaporean",
                                           "male", "this is enough!$%&*</>");
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_EDITED);
 
         ______TS("Typical case: changing genders for complete coverage");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "American",
-                                         "other", "this is enough!$%&*</>");
+        profilePage.editProfileThroughUi("short.name", "e@email.tmt", "inst", "American", "other",
+                                         "this is enough!$%&*</>");
         profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "American",
                                           "other", "this is enough!$%&*</>");
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "American",
-                                        "female", "this is enough!$%&*</>");
+        profilePage.editProfileThroughUi("short.name", "e@email.tmt", "inst", "American", "female",
+                                        "this is enough!$%&*</>");
```

Comment:
Do align the start of this line with the start of the params in the line above, i.e.
```
profilePage.editProfileThroughUi("short.name", ...
                                 "this is ...
```